Source/TrackManagerUI.cpp

```
@@ -1823,15 +1976,64 @@ void TrackManagerUI::onUpdate(double deltaTime) {
         }
     }
 
+    // === EDGE-SCROLLING DURING CLIP DRAG ===
+    // When dragging a clip near the edges of the grid, auto-scroll the view
+    if (m_isDraggingClipInstant && m_pixelsPerBeat > 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)